kyaml/setters2/add.go

```diff
@@ -145,6 +145,9 @@ type SetterDefinition struct {
 	// ListValues are the value of a list setter.
 	ListValues []string `yaml:"listValues,omitempty"`
 
+	// IsSet indicates the specified field has been explicitly assigned.
+	IsSet string `yaml:"isSet,omitempty"`
```

Comment:
You can make it a bool similar to `Required` field, so that you can avoid parsing string to bool above.

Reply:
Done, thank you for this bit.